### DISCUSSION TITLE
[Question][Hw2] Stability of the model generated by Z3

### DISCUSSION THREAD

**Discussion by Lee-Janggun** [prosyslab-classroom/cs424-program-reasoning#367] [Q&A]
- For my implementation of `eq.ml`, it passes `eq3.scm`, but the output is of different order than `scm.expected`.
  
```
satisfiable
 (define-fun tgt_x () Int
   2)
-(define-fun src_x () Int
-  2)
 (define-fun src_y () Int
   1)
+(define-fun src_x () Int
+  2)
```
- Is this still a valid result?
- In general, will our solution only be graded getting the satisfiability correct, or will we also be checked on the produced model?

**Comment by sujin0529** [comment]
Hi,

- Your result is also correct because only the output order is different.
- We will only evaluate on whether satisfiability is correct.

Thanks.

**Comment by Lee-Janggun** [comment]
So I can consider the outputs there mostly for debugging purposes. Thank you for the clarification!

**Comment by Lee-Janggun** [comment]
I also realize this is a duplicate of https://github.com/prosyslab-classroom/cs492-program-reasoning/issues/47... sorry for the inconvenience.
